File: Java/Grass.java
public class Grass extends Plant {
  public Grass(World world, Point position) {
    super(world, position, Constants.SYMBOL_GRASS, Constants.STRENGTH_GRASS, Constants.COLOR_GRASS, Constants.NAME_GRASS);
  }

  @Override
  public Organism Clone(Point position) {
    return new Grass(this.world, position);
  }
}

File: C++/Grass.cpp
#include "CONSTANTS.h"
#include "Grass.h"

Grass::Grass(World &world, const Point &position) : Plant(world, position, GRASS_SYMBOL, GRASS_STRENGTH, GRASS_COLOR, GRASS_NAME) {}

Organism* Grass::Clone(const Point &position) {
  return new Grass(this->world, position);
}